ls1intum/Artemis
# [Bugfix] Exam Instructor Fixes

### Checklist

### Motivation and Context
Fixes 3 bugs for instructors in exam mode:
 - This translation was broken ![grafik](https://user-images.githubusercontent.com/61357727/125347994-f234ae80-e35b-11eb-86ef-4b33156a54df.png)
 - The check for mandatory/optional exercise groups was not handling all cases correctly
 - Editing example submissions for text and modelling exercises navigated to an url for a regular exercise, removing the exam from the breadcrumbs

### Description
 - Fixed the translation
 - The exercise group check now works as follows: The number of exercises must be higher or equal to the number of mandatory exercise groups, but lower or equal to all exercise groups (including optional ones)
 - Example submissions for modeling and text exercises are now opened in the exam context, also added the missing route for modelling exercises

### Steps for Testing

1. Verify the total amount of points heading is shown correctly in both languages (at the exam detail page around the bottom):
![grafik](https://user-images.githubusercontent.com/61357727/125364859-83fbe600-e373-11eb-93f0-cd6c762f825d.png)
2. Verify the check for the amount of exercises is working correctly:
    - If there are less exercises groups than the amount of exercises, it should fail
    - If there are more mandatory exercise groups than the amount of exercises, it should fail
    - Otherwise it should work
    - The description should change depending on whether optional exercise groups are present
3. Create, edit, view, delete example submissions for exam text and modeling exercises, both should be opened in the exam context (the exam should be present both in the url and the breadcrumbs) and still work as expected

### Test Coverage
Not notably changed
